server/liveblog/posts/posts.py

```diff
@@ -129,8 +133,13 @@ def get(self, req, lookup):
         if lookup.get('blog_id'):
             lookup['blog'] = ObjectId(lookup['blog_id'])
             del lookup['blog_id']
+        logged_in_user = getattr(flask.g, 'user', None)
         docs = super().get(req, lookup)
         for doc in docs:
+            user_id = logged_in_user.get('_id')
+            creator_id = ObjectId(doc.get('original_creator'))
+            if creator_id == user_id and doc['post_status'] == 'draft':
+                doc['visible'] = True
```

Comment:
As it is right now, if you create a draft post and fetch the posts you will get your post having "visible" set to true. If you log in with another account and fetch the posts, you will get the draft post created by the other user but it will have "visible" set to false